models/user.go

```diff
@@ -1,3 +1,17 @@
-package models 
+package models
 
-type UserId int64
\ No newline at end of file
+import "strings"
+
+type UserId int64
+
+type EmailAddress struct {
+	emailAsString string
+}
+
+func NewEmailAddress(emailAddr string) *EmailAddress {
```

Comment:
why `emailAddr`? would prefer a long variable name

Reply:
I'll do emailAddressAsString then. Didn't want to use emailAddress lower case, especially considering its not an instance of the EmailAddress the type.